src/CSET/operators/read.py

```diff
@@ -488,6 +489,35 @@ def _fix_um_radtime_prehour(cube: iris.cube.Cube):
         pass
 
 
+def _fix_lfric_radtime_prehour(cube: iris.cube.Cube):
+    """Fix radiation which is output 1 minute before every hour."""
+    try:
+        # Check LFRic.
+        if "time_origin" in cube.coord("time").attributes:
+            if cube.name() in [
+                "toa_upward_longwave_flux_radiative_timestep",
+                "surface_net_longwave_flux_radiative_timestep",
+            ]:
```

Comment:
```suggestion
        if ("time_origin" in cube.coord("time").attributes and
            cube.name() in [
                "toa_upward_longwave_flux_radiative_timestep",
                "surface_net_longwave_flux_radiative_timestep",
            ]):
```
Use `and` rather than nested if statements.